i18n/usage/fr.txt

```diff
@@ -0,0 +1,22 @@
+{bold}{yellow}Utilisation{/yellow}{/bold}
+
+  {bold}{green}{appname}{/green}{/bold}
+    Affiche un menu pour choisir interactivement un exercice.
```

Comment:
"Affiche un menu interactif pour choisir un exercice." _Participe présent_ really looks ugly and unnatural.